presto-main/src/main/java/com/facebook/presto/SystemSessionProperties.java

```diff
@@ -170,6 +170,7 @@
     public static final String FRAGMENT_RESULT_CACHING_ENABLED = "fragment_result_caching_enabled";
     public static final String LEGACY_TYPE_COERCION_WARNING_ENABLED = "legacy_type_coercion_warning_enabled";
     public static final String INLINE_SQL_FUNCTIONS = "inline_sql_functions";
+    public static final String CHECK_ACCESS_CONTROL_ON_IMPACTING_COLUMNS_ONLY = "check_access_control_on_impacting_columns_only";
```

Comment:
The word "impacting" is quite confusing to me. Does this have a specific meaning in SQL / language / compiler? I felt "referenced" makes more sense.

Reply:
I couldn't think of a better word. I didn't like "referenced" because a column could be "referenced" if it appears in a subquery, even if it might be pruned later. And this definition of "referenced" is what is used in the codebase, for example, in "tableColumnReferences" in analysis, or "columnReferences". What do you think of "used columns" or "utilized columns"? Other ideas: "columns post pruning", "columns affecting output", "influencing columns"?

Reply:
ya, my worry is that "reference" has a specific meaning. Maybe a more generic "used"?

Reply:
Changed "impacting" to "utilized"